src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,364 @@
+    border-bottom: var(--pf-c-sequence__control--active--BorderBottomHeight) solid var(--pf-c-sequence__control--active--BorderBottomColor);
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-top: var(--pf-c-sequence__control--focus--BorderTopHeight) solid var(--pf-c-sequence__control--focus--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--focus--BorderBottomHeight) solid var(--pf-c-sequence__control--focus--BorderBottomColor);
+  }
+}
+
+.pf-c-sequence__control-left {
+  left: var(--pf-c-sequence__control-left--PositionLeft);
+  background-color: var(--pf-c-sequence__control-left--BackgroundColor);
+  border-right: var(--pf-c-sequence__control-left--BorderRightWidth) solid var(--pf-c-sequence__control-left--BorderRightColor);
+}
+
+.pf-c-sequence__control-right {
+  right: var(--pf-c-sequence__control-right--PositionRight);
+  background-color: var(--pf-c-sequence__control-right--BackgroundColor);
+  border-left: var(--pf-c-sequence__control-right--BorderLeftWidth) solid var(--pf-c-sequence__control-right--BorderLeftColor);
+}
+
+.pf-c-sequence__control-icon {
+  color: var(--pf-c-sequence__control-icon--Color);
+}
+
+// Horizontal list
+.pf-c-sequence__list {
+  display: inline-flex;
+  overflow: scroll;
+}
+
+// Change default styles for the list and list items
+.pf-c-sequence__list .pf-c-sequence__item {
+  list-style-type: none;
+
+  // added so that the list will be read by screen reader
+  &::before {
+    width: 1px;
+    content: "";
+  }
+}
+
+// Items inside of list
+.pf-c-sequence__item {
+  max-width: var(--pf-c-sequence__item--MaxWidth);
+  margin-right: var(--pf-c-sequence__item--MarginRight);
+  margin-left: var(--pf-c-sequence__item--MarginLeft);
+}
+
+// Vertically centers number and border
+.pf-c-sequence__number-container {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+// Sequence Link Position Number
+.pf-c-sequence__number {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-width: var(--pf-c-sequence__number--MinWidth);
+  max-height: var(--pf-c-sequence__number--MaxHeight);
+  padding: var(--pf-c-sequence__number--Padding);
+  margin-right: var(--pf-c-sequence__number--MarginRight);
+  font-size: var(--pf-c-sequence__number--FontSize);
+  color: var(--pf-c-sequence__number--Color);
+  text-align: center;
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border: var(--pf-c-sequence__number--BorderWeight) solid var(--pf-c-sequence__number--BorderColor);
+    border-radius: var(--pf-c-sequence__number--BorderRadius);
+  }
+}
+
+// Content
+.pf-c-sequence__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: var(--pf-c-sequence__content--MinHeight);
+}
+
+// Content Title
+.pf-c-sequence__content-title {
+  display: block;
```

Comment:
I don't think this is necessary since the parent is `flex`

Reply:
removed :)